### PR TITLE
add adr-22

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: docs/ADR-22-quests-progress-ui.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- Every quest is a regular scene that runs in a "global" scene. "global" means without limits of parcels.
+
+#### The quest (the SDK scene running the logic and visible part of the quest)
+
+- The quest' scene will use an SDK `QuestTrackingInformation(steps & progression)` primitive to consistently render the quest UI
+
+- The quest' scene will handle the execution of the logic of the quest
+
+- The progression of the quest will have to be saved and loaded from the quest'scene
```
Comment (BrianAmadori) [inline docs/ADR-22-quests-progress-ui.md:23]: A bit of clarification on "saved and loaded"? The quests persistence isn't going to be handled by the server?

File: docs/ADR-22-quests-progress-ui.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+![resources/ADR-22/quests-progress-ui-1-2.svg](resources/ADR-22/quests-progress-ui-1-2.svg)
+
+- It also allows any scene to show quests progression, enabling developers to perform their own quests in their LAND or wearables.
+- In this scenario, the quests state would be sent to unity using an SDK component, leveraging our current messaging structure. It can be attached to the `engine.rootEntity` to not treat it differently than other components.
+- Quests UI will be decoupled from the quests controller and quests server, aligning the implementation with our decentralization objectives (by removing a requirement of a centralized server).
+- Even though it is relly useful to test and debug quests, more limitations can be set in place for production mode.
+
+## Implementation details
+
+The renderer will receive an entity with a component of the type `QuestTrackingInformation(data)`. Based on that component, the renderer will have custom logic to render the component following the UX designs.
```
Comment (BrianAmadori) [inline docs/ADR-22-quests-progress-ui.md:84]: I don't think we will end going for this approach. The quest progress UI lifecycle will be very detached from the lifecycle of the SDK components. We are aiming for a data centric approach with more direct messages, more akin to our approach for chat and friends. 

@pablitar already designed and implemented the quest client to go towards this direction.

File: docs/ADR-22-quests-progress-ui.md (hunk 1 above)
Comment (menduz) [reply docs/ADR-22-quests-progress-ui.md:23]: That was the raw notes of the previous meeting. I can't recall correctly that part

File: docs/ADR-22-quests-progress-ui.md (hunk 2 above)
Comment (menduz) [reply docs/ADR-22-quests-progress-ui.md:84]: See the reasons

File: docs/ADR-22-quests-progress-ui.md (hunk 1 above)
Comment (pravusjif) [reply docs/ADR-22-quests-progress-ui.md:23]: That part means that the scene, using our SDK, would read-write the quest progress, similar to using a regular game achievements API to report progress.

**Pull request merged**